### PR TITLE
small corrections to MakeFile

### PR DESCRIPTION
**Pull request by Mingling94**
the rm default_app should be a mv to rename it to app... I think.
@DavidVorick comment on this, it deals with your most recent commit 16 days ago but I had to do it to run test-ui. Haven't touched MakeFile manually before in my life yo

### CODE REVIEW

**Comment by DavidVorick** [comment]
no it's supposed to remove it.

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ clean:
 
 dependencies:
 	npm install electron-prebuilt
-	rm node_modules/electron-prebuilt/dist/resources/default_app
```
Comment (Mingling94) [inline Makefile:8]: @DavidVorick, So at the very least, this needs a -rf flag.

Then a subsequent `make` yields: 
selection_007

**Comment by DavidVorick** [comment]
does it need the 'f'? I guess I did forget the 'r'.

You could run `@rm -r ...`, which supresses the output, but I think `rm -rf` is the better choice.

Thanks for catching that.

File: site/main.js [hunk 2]
```diff
@@ -17,10 +18,15 @@ app.on('ready', function() {
...
 		"height": 720,
 		"width": 1200,
+        "min-width": 800,
+        "min-height": 600,
 		"title": "Sia"
+
 	});
 	mainWindow.loadUrl('file://' + __dirname + '/html/index.html');
 
+	mainWindow.openDevTools();
```
Comment (DavidVorick) [inline site/main.js]: don't have this enabled in master.

File: site/main.js [hunk 3]
```diff
@@ -17,10 +18,15 @@ app.on('ready', function() {
 	mainWindow = new BrowserWindow({
 		"height": 720,
 		"width": 1200,
+        "min-width": 800,
+        "min-height": 600,
```
Comment (DavidVorick) [inline site/main.js]: tabbing

Comment (Mingling94) [inline site/main.js]: Heh, on my atom editor these appeared to be in line

**Comment by DavidVorick** [comment]
We should get a linter of some sort for javascript, something that works similarly to go fmt.

**Pull request merged**